pyds9/pyds9.py

```diff
@@ -13,7 +13,7 @@
 
 from collections import defaultdict
 import contextlib
-from distutils.spawn import find_executable
+import shutil
```

Comment:
@olebole if you could replace this with

    try:
        from shutil import which
    except ImportError
        from distutils.spawn import find_executable as which

and in the lines below use only ``which``, I can merge your PR and fix/update the Travis build in a different PR